build/ci.bazelrc

```diff
@@ -58,6 +58,7 @@ build:ci-linux-common --action_env=CC=/usr/lib/llvm-19/bin/clang
 build:ci-linux-common --host_action_env=CC=/usr/lib/llvm-19/bin/clang
 
 build:ci-linux --config=ci-linux-common --remote_download_regex=".*src/workerd/server/workerd.*"
+build:ci-linux --build_tag_filters=-off-by-default,+requires-container-engine
```

Comment:
You should update test_tag_filters instead – otherwise the test is being built and still filtered out. All targets that are test targets or their dependencies will be built if they are not excluded by the test filter, even if they would be filtered out by the build filter.
However, even with test_tag_filters we'd run into trouble: If we use the filter as written here we would only run the container test and no other test since "Only those test targets will be found that contain at least one included tag and do not contain any excluded tags". https://bazel.build/reference/command-line-reference#build-flag--test_tag_filters

Reply:
Two ways to address this:
1) Mark the test as ["off-by-default", "requires-container-engine"] as written. Would require running `bazel test` several times, the first time with -off-by-default filter as before and the second time with +requires-container-engine
2) Mark test as just ["requires-container-engine"]. Avoids having to run Bazel twice, but would require adding -requires-container-engine to all filters except for the one used when we want to run this test, including the ones downstream.